apps.yml

```diff
@@ -1017,10 +1017,7 @@ apps:
     - /iam-grafana
 - application:
     authorized_groups:
-    - mozilliansorg_nda
     - mozilliansorg_boxcom-access
-    - team_moco
-    - team_mofo
     - team_office_support
 #    - team_mzla
```

Comment:
I agree. `team_mzla` should be removed from here completely since access it covered explicitly by `mozilliansorg_boxcom-access`